src/components/DraggableViewSection.tsx

```diff
@@ -14,17 +21,33 @@ export interface DraggableViewSectionProps {
   onDimensionChange: (change: number) => void;
 }
 
-function DraggableViewSection({ children, offset, dimension, vertical, dimensionsComputed, isLast, onDimensionChange }: DraggableViewSectionProps) {
+function DraggableViewSection({
+  children,
+  offset,
+  dimension,
+  vertical,
+  dimensionsComputed,
+  isLast,
+  onDimensionChange,
+}: DraggableViewSectionProps) {
+  const dragEvent = useRef<DragEvent<Body> | null>(null);
```

Comment:
You can just drop the explicit initial null value here and use 
```const dragEvent = useRef<DragEvent<Body>>();```
Then in `handleDragEnd` you'd set `dragEvent.current = undefined`